handler/create_user.go

```diff
@@ -0,0 +1,65 @@
+package handler
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"os"
+
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
+)
+
+func CreateUser(w http.ResponseWriter, r *http.Request) {
+	db, err := sqlx.Connect("postgres", os.Getenv("POSTGRES_DSN"))
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
```

Comment:
why are you connecting to the database on every request? if you read sqlx documentation, you should share a database client for all your requests. as well, connecting to the database is several roudtrips so you are degrading latency